scss/_base_tables.scss

```diff
@@ -45,7 +45,7 @@
     }
 
     tr {
-      border-bottom: 1px solid $colors--theme--border-default;
+      border-bottom: $input-border-thickness solid $colors--theme--border-default;
```

Comment:
@lyubomir-popov Is this OK to use `$input-border-thickness` for tables, or is it reserved to other (interactive) elements?